start.sh

```diff
@@ -0,0 +1,80 @@
+#!/bin/sh
+set -eu
+
+USERNAME=$1
+INSTALLER_IMAGE="registry.svc.ci.openshift.org/openshift/origin-v4.0:installer"
+ANSIBLE_IMAGE="quay.io/vrutkovs/openshift-40-centos"
+
+if [ -z ${USERNAME} ]; then
+  echo "Usage: ./run.sh <username>"
+  exit 1
+fi
+
+if [ ! -f ./pull_secret.json ]; then
+  echo "Pull secret not found!"
+  exit 1
+fi
+
+if [ ! -d injected ]; then
+  echo "injected directory not found!"
+  exit 1
+fi
+
+echo "Removing existing artifacts"
+rm -rvf install-config.yml .openshift_install_state.json .openshift_install.log *.ign || true
+
+echo
+echo "Fetching installer"
+podman pull ${INSTALLER_IMAGE}
+podman run -rm -ti ${INSTALLER_IMAGE} version
```

Comment:
Need `sudo` on these lines too.

Reply:
this now can be changed via updating `PODMAN`